components/eamxx/docs/user/index.md

```diff
@@ -1,3 +1,9 @@
 # SCREAM User Guide
 
+This section contains documentation on how to create, setup, and run CIME cases with EAMxx as the atmosphere component.
+It is assumed that the reader has a familiarity with [CIME case
+control system](https://esmci.github.io/cime/versions/master/html/users_guide/index.html). In particular, we assume
+that the user knows how to create a case, and what the `case.setup`, `case.build`, and `case.submit` commands do.
+
+
 For the time being, see our [public confluence EAMxx user guide](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3858890786/EAMxx+User+s+Guide)
```

Comment:
Maybe change to "This guide is under construction. Until these pages are complete, you may find earlier documentation at [public confluence EAMxx user guide](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3858890786/EAMxx+User+s+Guide) to be useful."?